quickstarts/kind/index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Kubernes Kind Quickstart
+layout: default
+---
+
+Kubernetes Kind is a single-node Kubernetes cluster implemented as a single Docker image that runs as a container.
```

Comment:
My understanding was that Kid could do multinode clusters as well. Is it not the case?

Reply:
You're right. I'll change the sentence.